sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_common/_base_prompty_eval.py

```diff
@@ -74,3 +78,9 @@ async def _do_eval(self, eval_input: Dict) -> Dict[str, float]:
             if match:
                 score = float(match.group())
         return {self._result_key: float(score)}
+    
+    @override
+    def __call__(self, *, query=None, response=None, context=None, conversation=None, **kwargs):
+        result = super().__call__(query=query, response=response, context=context, conversation=conversation, **kwargs)
+        result = update_with_passing_label(result=result, passing_score=self._passing_score, metric_name=self._result_key)
```

Comment:
Would it make more sense for this to live in the `_do_eval` method in `_base_prompt_eval.py`?

Reply:
Ahh - good point. The other evaluators don't have a `do_eval` and do everything in `__call__`, so I didn't notice how `PromptyBaseEvaluator`'s structure is slightly different. I'll move it.